Client/Frontend/Home/FirefoxHomeViewController.swift

```diff
@@ -315,14 +328,10 @@ extension FirefoxHomeViewController {
             switch self {
             case .pocket:
                 var numItems: CGFloat = FirefoxHomeUX.numberOfItemsPerRowForSizeClassIpad[traits.horizontalSizeClass]
-                if UIApplication.shared.statusBarOrientation.isPortrait {
-                    numItems = numItems - 1
-                }
-                if traits.horizontalSizeClass == .compact && UIApplication.shared.statusBarOrientation.isLandscape {
-                    numItems = numItems - 1
-                }
+                numItems = numItems - 1
+                
                 return numItems
-            case .topSites, .libraryShortcuts:
+            case .topSites, .libraryShortcuts, .recentlySaved: // TODO FIGURE THIS OUT
```

Comment:
Need some advice here!

In general, I'm looking for a clean way to skip adding this section entirely if it doesn't meet conditions (when no bookmarks exist, when it's an iPad etc.).